misc/requirements/requirements-pylint.txt

```diff
@@ -4,7 +4,6 @@ astroid==1.5.3
 certifi==2017.7.27.1
 chardet==3.0.4
 github3.py==0.9.6
-idna==2.6
```

Comment:
How would I go about it if I needed to use a package that's not in the requirements? Say, in my case, I wanted to use `requests` package in my tests.